chemprop/nn/loss.py

```diff
@@ -116,7 +116,7 @@ class MVELoss(LossFunction):
     """
 
     def _calc_unreduced_loss(self, preds: Tensor, targets: Tensor, *args) -> Tensor:
-        mean, var = torch.chunk(preds, 2, 1)
+        mean, var = torch.unbind(preds, dim=-1)
```

Comment:
```suggestion
        mean, var = torch.unbind(preds, dim=-1)
        mean, var = mean.view(-1, 1), var.view(-1, 1)
```
Same thing to the other `unbind` changes. Using `unbind` drops a dimension, which we need to add back because the shape of `mean` needs to match the shape of `targets` or else you get unwanted broadcasting:
```
targets = torch.tensor([0., 1.]).view(-1, 1)
preds = torch.tensor([[0., 1., 2., 3.]]).view(-1, 2)

mean, var = torch.unbind(preds, dim=-1)
mean.shape is torch.Size([2])
(mean - targets).shape is torch.Size([2,2])

mean, var = torch.chunk(preds, 2, 1)
mean.shape is torch.Size([2, 1])
(mean - targets).shape is torch.Size([2,1])
```
This isn't caught by our tests because `test_MVE` in `test_loss_functions` uses a single datapoint (batch size = 1) and the broadcasting works. We should probably improve our testing of the loss functions/metrics, which I can do in my metrics PR.

Reply:
shouldn't `preds` have shape $(b, t, u)$ from the FFN? So `mean`, `var`, etc. would then have shape $(b, t)$, which is the same shape as `targets`

Reply:
That's correct, I forgot we added a dimension.